test/post-911-gib-status/01-authed.e2e.spec.js

```diff
@@ -0,0 +1,48 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts.js');
+const GibsHelpers = require('../e2e/post-911-gib-status-helpers.js');
+const LoginHelpers = require('../e2e/login-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const token = LoginHelpers.getUserToken();
+
+    GibsHelpers.initApplicationMock(token);
+
+    // Ensure main status page renders.
+    LoginHelpers.logIn(token, client, '/education/gi-bill/post-9-11/ch-33-benefit', 3)
+      .waitForElementVisible('body', Timeouts.normal)
+      .axeCheck('.main')
+      .assert.title('Check Benefit: Vets.gov')
+      .waitForElementVisible('.schemaform-title', Timeouts.slow);  // First render of React may be slow.
+
+    // Checking field in UserInfoSection has rendered
+    client.expect.element('#fullName').text.to.contain('First Last');
+
+    // Checking field in EnrollmentHistory has rendered
+    client.expect.element('#enrollment-0 h4').text.to.contain('11/01/2012 to 12/01/2012 at Purdue University');
+
+    // Click on print page
+    client
+      .click('#print-button')
+      .windowHandles((result) => {
+        const newWindow = result.value[1];
+        client.switchWindow(newWindow);
+      });
+
+    // Ensure print page renders.
+    // We need to log in again here because Electron loses the session
+    // when opening a new window.
+    LoginHelpers.logIn(token, client, '/education/gi-bill/post-9-11/ch-33-benefit/print', 3)
+      .waitForElementVisible('body', Timeouts.normal)
+      .axeCheck('.main')
+      .assert.title('Check Benefit: Vets.gov')
```

Comment:
Looks like this is where it fails on Jenkins:
```
 ✖ Testing if the page title equals "Check Benefit: Vets.gov".  - expected "Check Benefit: Vets.gov" but got: "Error"
    at /application/test/post-911-gib-status/01-authed.e2e.spec.js:39:15

```
2 ideas, though both could be red herring:

1) From reading the same pages you probably read about `windowHandles` it looks like sometimes people call `client.switchWindow()` and other times `this.switchWindow()`. Is there a difference?

2) I wonder if you put this entire block inside the callback passed to `windowHandles` how it would behave. Again, not sure whether to use `client` or `this`

Otherwise,  I'm stumped too....maybe time to send a plea to #vetsgov-fe pointing to the jenkins log and this PR